src/pages/patterns/save.mdx

```diff
@@ -0,0 +1,258 @@
+---
+title: Save
+description: >
+  The Save command is used whenever a user wants to apply changes or maintain progress.
+---
+
+## Anatomy
+
+### Action footer
+
+![Save pattern anatomy](figma:patterns-save-anatomy-action-footer.png)
+
+1. The **Save** button is the [primary button](/components/button#type) on a page, placed at the end
+   of all form inputs
+2. The **Cancel** button is a [basic button](/components/button#type), left of the Save button
+3. In a [page footer](/components/chrome#footer), the Save and Cancel buttons remain fixed
+
+### Saving changes
+
+Saving applies to a page or view.
+
+Always [enable](/patterns/buttons#disabled) the Save button for users to trigger an action even if
+there are no new changes to save.
+
+- If there are tabs, save all data per tab
+- If the page or tab has an indication of the last edit date, that timestamp should update
+- After changes are saved, show a loader inside the Save button if saving takes time, and ignore
+  trigger actions for the duration
+- Ensure a custom, translated label for the loader is provided on implementation
+
+import SavePatternInteraction from '../../examples/save/SavePatternInteraction.tsx';
+import SavePatternInteractionCode from '!!raw-loader!../../examples/save/SavePatternInteraction.tsx';
```

Comment:
I can do this but I think it's clearer using the words "Save" in this particular example. 

I might update the `ButtonPatternInteraction.tsx` example to use the words "Save" rather than "Complete purchase" to keep the guidance more generic. That solves the design problem across both patterns and utilizes the single-coded example.

Reply:
Ah, I missed the subtle difference. Thanks for the solve.